lib/services/project-changes-service.ts

```diff
@@ -300,5 +305,20 @@ export class ProjectChangesService implements IProjectChangesService {
 		}
 		return false;
 	}
+
+	private getAppFiles(appDirectoryPath: string): string[] {
+		return this.$fs.enumerateFilesInDirectorySync(appDirectoryPath, (filePath: string, stat: IFsStats) => filePath.indexOf(APP_RESOURCES_FOLDER_NAME) === -1);
```

Comment:
maybe it is better to ensure the `path.basename(filePath) !== appresources name`
You should not use the APP_RESOURCES_FOLDER_NAME constant here - you should read the value from `projectData` as by using .nsconfig the user might change directory's name.